libs/MeshKernel/src/Mesh2D.cpp

```diff
@@ -37,6 +37,8 @@
 
 using meshkernel::Mesh2D;
```

Comment:
I'd simply put the class def in `namespace meshkernel { .... }`.
This applies to several classes throughout the repo.

Reply:
I sis not use this form to avoid an extra level of nesting. But it could indeed be used